app/controllers/api_controller/reports.rb

```diff
@@ -13,5 +13,11 @@ def show_reports
       end
       show_generic(:reports)
     end
+
+    def run_resource_reports(_type, id, _data)
```

Comment:
i believe this method is called automagically with these args, so i can't change its method signature